Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/publish.py

```diff
@@ -61,7 +61,10 @@ def branches_on(cls, repository, ref):
             return {}
         result = collections.defaultdict(set)
         for line in output.stdout.splitlines():
-            split = line.lstrip().split('/', 2)
+            if line[0] == '*':
```

Comment:
We should filter for `+`, too, which indicates a branch checked out in another worktree. But can we just change the command to pass something like `--format=%(refname)`?